src/Stratis.Bitcoin.IntegrationTests/BlockStore/ProofOfWorkSpendingSteps.cs

```diff
@@ -82,7 +82,11 @@ private void spending_the_coins_from_original_block()
             try
             {
                 this.lastTransaction = this.sendingStratisBitcoinNode.FullNode.WalletTransactionHandler()
-                    .BuildTransaction(SharedSteps.CreateTransactionBuildContext(SendingWalletName, AccountName, WalletPassword, sendtoAddress.ScriptPubKey, Money.COIN * 1, FeeType.Medium, 101));
+                    .BuildTransaction(SharedSteps.CreateTransactionBuildContext(SendingWalletName
+                        , AccountName
```

Comment:
I find these `,` at the start of the new line a bit strange - probably doesn't break the policy but just looks slightly odd to me - like when you do sql statements so you can comment lines out etc, except you can't/wouldn't here - wouldn't end of prev line be more in line with most of the rest of codebase?